webpack.config.js

```diff
@@ -0,0 +1,24 @@
+var path = require('path');
+
+module.exports = {
+  mode: 'development',
+  entry: './client/index.jsx',
+  output: {
+    path: path.resolve(__dirname, 'public'),
+    filename: 'main.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.m?jsx$/,
```

Comment:
I don't think you're using any `.mjs` files; you should omit that from your regex.